client/lib/abtest/active-tests.js

```diff
@@ -61,4 +61,12 @@ module.exports = {
 		},
 		defaultVariation: 'notOffered'
 	},
+	monthlyPlanPricing: {
+		datestamp: '20160118',
```

Comment:
usually I prefer to set the real date in the PR, then simply change the date locally just for testing purpose. But I'm open to a different approach, if you think it's better. The starting date is important and I want to make sure it's the right one when merged. It can be set right from the beginning, or at the end, I'm fine with defining a best practice and stick with it.